src/Polytopes/Cone.jl

```diff
@@ -9,17 +15,64 @@ struct Cone #a real polymake polyhedron
 end
 function Cone(Rays::Union{Oscar.MatElem,AbstractMatrix})
     Cone(Polymake.polytope.Cone{Polymake.Rational}(
-        RAYS = matrix_for_polymake(Rays),
+        INPUT_RAYS = matrix_for_polymake(Rays),
     ))
 end
+function Cone(Rays::Union{Oscar.MatElem,AbstractMatrix}, LS::Union{Oscar.MatElem,AbstractMatrix}; non_redundant::Bool=false)
+   if non_redundant
+       Cone(Polymake.polytope.Cone{Polymake.Rational}(
```

Comment:
It would be really nice to have some more tests -- notice the Codecov coverage comments here, which indicate a lot of code that is never exercised by the test suite.